docker_explorer/explorer.py

```diff
@@ -202,15 +202,14 @@ def GetContainersJson(self, only_running=False):
       if container_object.mount_id:
         container_json['mount_id'] = container_object.mount_id
 
-      if container_object.mount_points:
-        container_json['mount_points'] = {}
-        for mount_point, details in container_object.mount_points.items():
+      mount_points = container_object.GetMountpoints()
+      if mount_points:
+        container_json['mount_points'] = []
+        for source, mount_point in mount_points:
           d = collections.OrderedDict()
-          d['type'] = details.get('Type')
-          d['mount_point'] = mount_point
-          d['source'] = details.get('Source')
-          d['RW'] = details.get('RW')
-          container_json['mount_points'][mount_point] = d
+          d['source'] = os.path.join(self.docker_directory, source)
+          d['destination'] = os.path.join(os.path.sep, mount_point)
+          container_json['mount_points'].append(d)
```

Comment:
you can use `.setdefault('mount_points', []).append(d)` here and get rid of the parent if condition and the 'mount_point' value intialization: 
```
if mount_points:
        for mount_point, details in container_object.mount_points.items():	        container_json['mount_points'] = []
```

Reply:
done